kolibri/plugins/device_management/assets/src/views/SelectContentPage/SelectedResourcesSize.vue

```diff
@@ -99,8 +106,9 @@
       export: 'export',
       import: 'import',
       notEnoughSpace: 'Not enough space on your device. Select less content to make more space.',
-      remainingSpace: 'Your remaining space: {space}',
-      resourcesSelected: 'Resources selected: {resources, number, integer} ({fileSize})',
+      availableSpace: 'Space available: {space}',
```

Comment:
Could this be "Free disk space"? (I think it's a more conventional term)

Reply:
Oh... we should perhaps avoid to say "disk" because it may be a "card" / "drive" / "partition". In that case, "Free drive space" is more universal: On Windows "drive" is very to the point, and on Linux/Mac, I think it's understandable both for partitions, flash cards etc.